packages/app-mobile/components/NoteEditor/CodeMirror/markdownCommands.ts

```diff
@@ -0,0 +1,440 @@
+// CodeMirror 6 commands that modify markdown formatting (e.g. toggleBold).
+
+import { EditorView, Command } from '@codemirror/view';
+
+import { ListType } from '../types';
+import {
+	SelectionRange, EditorSelection, ChangeSpec, Line, TransactionSpec,
+} from '@codemirror/state';
+import { getIndentUnit, indentString, syntaxTree } from '@codemirror/language';
+import {
+	RegionSpec, growSelectionToNode, renumberList,
+	toggleInlineFormatGlobally, toggleRegionFormatGlobally, toggleSelectedLinesStartWith,
+	isIndentationEquivalent, stripBlockquote, tabsToSpaces,
+} from './markdownReformatter';
+
+const startingSpaceRegex = /^(\s*)/;
+
+export const toggleBolded: Command = (view: EditorView): boolean => {
+	const spec = RegionSpec.of({ template: '**', nodeName: 'StrongEmphasis' });
+	const changes = toggleInlineFormatGlobally(view.state, spec);
+
+	view.dispatch(changes);
+	return true;
+};
+
+export const toggleItalicized: Command = (view: EditorView): boolean => {
+	const changes = toggleInlineFormatGlobally(view.state, {
+		nodeName: 'Emphasis',
+
+		template: { start: '_', end: '_' },
```

Comment:
Should we really be using `_` here? The desktop app buttons only use `*` to add emphasis and bold emphasis - the experience should be consistent.